test/unit/sagas/clients.js

```diff
@@ -119,7 +119,7 @@ test('export repo', (t) => {
         url,
         exportType,
         scenario.project.projectKey,
-        {name},
+        {repoName: name, username: undefined},
```

Comment:
Not great, but the username is determined inside a function from the state and I couldn't (quickly) figure out a way to inject it. This at least tests that the key is correctly produced 🤷‍♀️